JM/CF/Debugger/layouts/Debugger_Block.layout

```diff
@@ -0,0 +1,57 @@
+WrapSpacerWidgetClass BlockSpacer {
+ ignorepointer 1
+ position 0 0
+ size 500 1
+ hexactpos 0
+ vexactpos 0
+ hexactsize 1
+ vexactsize 0
+ Padding 0
+ Margin 0
+ "Size To Content H" 1
+ {
+  MultilineTextWidgetClass EntryKeys {
+   ignorepointer 1
+   position 0 0
+   size 0.4 1
+   hexactpos 0
+   vexactpos 0
+   hexactsize 0
+   vexactsize 0
+   scriptclass "ViewBinding"
+   text "AAA"
+   font "gui/fonts/system"
+   "exact text" 1
+   "size to text h" 1
+   "size to text v" 1
+   {
+    ScriptParamsClass {
+     Binding_Name "EntryKeys"
+    }
+   }
+  }
+  MultilineTextWidgetClass EntryValues {
+   ignorepointer 1
+   position 0.4 0
+   size 0.6 0.98
+   hexactpos 0
+   vexactpos 0
+   hexactsize 0
+   vexactsize 0
+   scriptclass "ViewBinding"
+   text "deserunt mollit anim id est laborum.\""\
```

Comment:
Can we remove this sample text for the release and just replace it with a short "PLACEHOLDER"?